src/src/com/microsoft/aad/adal/StorageHelper.java

```diff
@@ -127,7 +127,7 @@
 
     private static String sBlobVersion;
```

Comment:
should this also be non static? what if it fails to get key and macKey for one StorageHelper instance, but won't fail for another?